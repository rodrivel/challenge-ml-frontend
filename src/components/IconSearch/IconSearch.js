import React from 'react';
import iconSearch from '../../assets/ic_Search.png';
import iconSearch2x from '../../assets/ic_Search@2x.png.png';

const IconSearch = () => <img width={18} src={iconSearch2x} srcSet={`${iconSearch}18w, ${iconSearch2x} 36w`} alt="Icon buscar"/>;

export default IconSearch;