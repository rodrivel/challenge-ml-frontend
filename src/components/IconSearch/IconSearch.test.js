import React from 'react';
import { shallow } from 'enzyme';
import IconSearch from './IconSearch';

it('expect to render IconSearch component', () => {
  expect(shallow(<IconSearch />)).toMatchSnapshot();
});
