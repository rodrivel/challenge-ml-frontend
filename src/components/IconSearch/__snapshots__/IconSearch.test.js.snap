// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render IconSearch component 1`] = `ShallowWrapper {}`;
